packages/manager/apps/pci-billing/src/pages/billing/history/History.page.tsx

```diff
@@ -67,8 +67,8 @@ export default function History() {
       ) : (
         <>
           <HistoryResume
-            totalPrice={consumption?.totals.total}
-            isPostPaidUsageBilling={isPostPaidUsageBilling}
+            totalPrice={consumption?.totals.total ?? 0}
+            isPostPaidUsageBilling={isPostPaidUsageBilling ?? false}
```

Comment:
It should be added to the constant the same way it's done for isTrustedZone for more clarity